packaging/linux/rpm/template.go

```diff
@@ -829,7 +865,13 @@ func (w *specWrapper) Files() fmt.Stringer {
 	}
 
 	for _, l := range artifacts.Links {
-		fmt.Fprintln(b, l.Dest)
+		if l.UID != 0 || l.GID != 0 {
+			// Use %attr to specify ownership, with "-" for mode to preserve symlink's mode
+			fmt.Fprintf(b, "%%attr(-, %d, %d) %s\n", l.UID, l.GID, l.Dest)
```

Comment:
`%attr` does not support numeric id's for user and group.
I think we'll need to make these strings (in the spec) and expect people to put names in rather than ID's.